app/views/admin/facility_groups/_form.html.erb

```diff
@@ -1,6 +1,11 @@
 <div class="row">
   <div class="col-md-6">
     <%= bootstrap_form_with(model: [:admin, facility_group], local: true, autocomplete: "off", label_errors: true, html: {id: "facility-group-form"}) do |form| %>
+      <% if @facility_group.errors.any? %>
+        <div id="errors">
+          <%= form.label "Saving the FacilityGroup failed, please fix the following errors: #{@facility_group.errors.full_messages.to_sentence}", class: "text-danger" %>
```

Comment:
👾  
```suggestion
          <%= form.label "Saving the Facility Group failed, please fix the following errors: #{@facility_group.errors.full_messages.to_sentence}", class: "text-danger" %>
```

PS What changed about this page?

Reply:
It wasn't showing errors properly when the Facility Group failed saving...I wanted to get the error message into the view _somehow_, and this was the simplest way to do it.